### PR TITLE
Fix `BottomPanel` excessive width.

### PR DESCRIPTION
**Pull request by WhalesState**
You can have as many BottomPanel plugins as you want!

https://github.com/user-attachments/assets/3b687255-0aa7-4f34-9ffd-b06789fa3e4c

### CODE REVIEW

**Review by Murly117 — APPROVED** [review]

File: editor/gui/editor_bottom_panel.cpp [hunk 1]
```diff
@@ -255,9 +297,35 @@ EditorBottomPanel::EditorBottomPanel() {
 	bottom_hbox->set_custom_minimum_size(Size2(0, 24 * EDSCALE)); // Adjust for the height of the "Expand Bottom Dock" icon.
 	item_vbox->add_child(bottom_hbox);
 
+	left_button = memnew(Button);
+	left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the begin.\nHold Shift to scroll one page."));
```
Comment (jss2a98aj) [inline editor/gui/editor_bottom_panel.cpp:301]: "Hold Ctrl to scroll to the begin." should probably be "Hold Ctrl to scroll to the begining." or "Hold Ctrl to scroll to the start."

**Review by jss2a98aj — CHANGES_REQUESTED** [review]
This look like it will be a really nice feature. One of the tooltips should probably be tweaked slightly, but otherwise I did not notice any issues.

**Review by jss2a98aj — APPROVED** [review]

**Comment by Starkium** [comment]
looks like there's a typo
`-	left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the begining.\nHold Shift to scroll one page."));
+	left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the beginning.\nHold Shift to scroll one page."));`

**Comment by jss2a98aj** [comment]
> looks like there's a typo
> 
> * left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the begining.\nHold Shift to scroll one page."));
> 
> * left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the beginning.\nHold Shift to scroll one page."));

That typo was my fault, I had it messed up when suggesting a change.

**Comment by Starkium** [comment]
someone stopped the tests. I'll rerun this again once those are done

**Comment by jss2a98aj** [comment]
> someone stopped the tests. I'll rerun this again once those are done

Those stopped automatically when the static check failed from the typo.

File: editor/gui/editor_bottom_panel.cpp [hunk 2]
```diff
@@ -255,9 +297,35 @@ EditorBottomPanel::EditorBottomPanel() {
 	bottom_hbox->set_custom_minimum_size(Size2(0, 24 * EDSCALE)); // Adjust for the height of the "Expand Bottom Dock" icon.
 	item_vbox->add_child(bottom_hbox);
 
+	left_button = memnew(Button);
+	left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the begining.\nHold Shift to scroll one page."));
```
Comment (Starkium) [inline editor/gui/editor_bottom_panel.cpp:301]: spell check linter "beginning"

**Review by jss2a98aj — APPROVED** [review]

**Pull request merged**